postgres-appliance/postgres_ha.sh

```diff
@@ -74,20 +74,6 @@ write_postgres_yaml
 
 write_archive_command_environment
 
-function noterm
-{
-	echo "Received TERM signal, but not doing anything"
-}
-
-# resurrect etcd if it's gone
-(
-  trap noterm SIGTERM
-  while true
-  do
-    etcd -name "proxy-$SCOPE" -proxy on  --data-dir=etcd -discovery-srv $ETCD_DISCOVERY_URL
```

Comment:
Proxy could be started by Taupage.
The only what we should do - put 'etcd_discovery_domain' into TaupageConfig
Not sure that this part is already functional, because there are some hardcoded ip's and TODO's:
https://github.com/zalando-stups/taupage/blob/master/runtime/etc/init/etcd.conf https://github.com/zalando-stups/taupage/blob/master/runtime/etc/init/register-in-etcd.conf